cfme/services/catalogs/myservice.py

```diff
@@ -44,15 +45,17 @@
 menu.nav.add_branch(
     'my_services',
     {
-        'service':
+        'all_service':
```

Comment:
I more meant and there are no other tests that try to force_navigate to "service" that will be broken after this merge?